.github/workflows/ci.yml

```diff
@@ -348,20 +348,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runtime: [io.containerd.runtime.v1.linux, io.containerd.runc.v1, io.containerd.runc.v2]
```

Comment:
Why are `io.containerd.runtime.v1.linux` and `io.containerd.runc.v1` removed?

Reply:
ref: https://github.com/containerd/containerd/pull/9502#issuecomment-1854383887 as part of making the CI green. Still have to see if there is a way to enable cgroupv1 in our runner and test the older shims

Reply:
cgroupv1 is disabled on the ubuntu 22 and ubuntu 24 runners. And since its on the kernel parameter to enable it, we cannot enable it in our runner machines. So will have to skip those tests similar to what was done in 1.7 branch